.github/workflows/ci.yml

```diff
@@ -278,15 +278,24 @@ jobs:
       - name: check no_std
         if: matrix.kind == 'no_std'
         shell: bash
+        # XXX TBD CHECK no-std for wgpu-types & wgpu-core in separate steps - ???
         run: |
           set -e
 
-          # check with no features
+          # check wgpu-types with no features
           cargo clippy --target ${{ matrix.target }} ${{ matrix.extra-flags }} -p wgpu-types --no-default-features
 
-          # Check with all features except "std".
+          # check wgpu-core with no features
+          cargo clippy --target ${{ matrix.target }} ${{ matrix.extra-flags }} -p wgpu-core --no-default-features
```

Comment:
Dang this is not failing the way I expected: https://github.com/gfx-rs/wgpu/actions/runs/12820839400/job/35751074262?pr=6926

The following command I copy-pasted from the build log reproduces on my macBook (I needed to do `rustup target add x86_64-unknown-none` first):

```sh
cargo clippy --target x86_64-unknown-none  -p wgpu-core --no-default-features
```

I would like to fix this, if possible.